README.md

```diff
@@ -2,7 +2,7 @@
 # passgo
 stores, retrieves, generates, and synchronizes passwords securely and is written in Go! It is inspired by https://passwordstore.org but has a few key differences. The most important difference is passgo is not GPG based. Instead it uses a master password to securely store your passwords.
 
-passgo is meant to be secure enough that you can publically post your password vault. I've started publishing my passwords [here](https://github.com/ejcx/passwords.git)
+passgo is meant to be secure enough that you can publicly post your password vault. I've started publishing my passwords [here](https://github.com/ejcx/passwords.git).
```

Comment:
```
The Columbia Guide to Standard English says there is no difference between the two words; they are just two different spellings, with "publicly" being the most common and accepted form. I imagine "publically" is used more often in law because it is a bigger word and looks more impressive.
```

Whatever. We aren't lawyers here. LGTM